packages/babel-types/test/builders/experimental/__snapshots__/classProperty.js.snap

```diff
@@ -0,0 +1,37 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
+
+exports[`builders experimental classProperty should validate 1`] = `
+Object {
+  "computed": false,
+  "decorators": null,
+  "key": Object {
+    "type": "StringLiteral",
+    "value": "test",
+  },
+  "static": null,
```

Comment:
Could `static` default to `false`? Like we did on `computed`
https://github.com/babel/babel/blob/fced5cea430cc00e916876b663a8d2a84a5dad1f/packages/babel-types/src/definitions/es2015.js#L409-L412

Reply:
It is defined as optional in `classMethodOrPropertyCommon` just above `computed`:
https://github.com/babel/babel/blob/fced5cea430cc00e916876b663a8d2a84a5dad1f/packages/babel-types/src/definitions/es2015.js#L405-L408

Does it have sense to change definitions there or rather fields in `ClassProperty` definition? Tests are passed in both cases, I think changing `classMethodOrPropertyCommon` has more sense.

Reply:
The default `static` was `false` but later changed to `optional` in #5856. Back at that time `static` class property are in staged-2. I believe `optional: true` is used to convey that `static` field is used by TypeScript only at that time.

Now as we have static field/method in standard, I think it make sense to set a default value for `static`.

Reply:
Yah, all that's needed is `static: { default: false }`. It should add `assertValueType("boolean")` automatically.

Reply:
I'll keep `assertValueType("boolean")` for consistency with the rest of the code.